src/Services/Service.js

```diff
@@ -41,6 +42,10 @@ export var Service = L.Evented.extend({
     return this;
   },
 
+  setTimeout: function (timeout) {
```

Comment:
Could you implement a companion `getTimeout` route? It is nice to have parity with the getters and setters.